cmd/clusterctl/clusterdeployer/bootstrap/minikube/minikube.go

```diff
@@ -61,7 +62,7 @@ var minikubeExec = func(env []string, args ...string) (string, error) {
 	cmdOut, err := cmd.CombinedOutput()
 	klog.V(2).Infof("Ran: %v %v Output: %v", executable, args, string(cmdOut))
 	if err != nil {
-		err = fmt.Errorf("error running command '%v %v': %v", executable, strings.Join(args, " "), err)
+		err = errors.Errorf("error running command '%v %v': %v", executable, strings.Join(args, " "), err)
```

Comment:
`Wrapf()` should be used in all places where there is an error wrapped, but also extra args.
https://github.com/pkg/errors/blob/master/errors.go#L198